plugin/cypress/e2e/featureFiles/navigation.feature

```diff
@@ -8,16 +8,21 @@ Feature: Kiali integration with OCP Console
   Background:
     Given user is logged as administrator in OCP Console
     And user clicks on the Service Mesh icon in the left navigation bar
+    And cypress intercept hooks are registred
 
   Scenario: Service mesh buttons are displayed
     Then buttons for Overview, Graph and Istio Config are displayed
 
   Scenario: Overview page is displayed correctly
-    Then user is redirected to the OSSMC "Overview" page
+    When user is redirected to the OSSMC "Overview" page
+    And kiali plugin is loaded
```

Comment:
I would add `kiali plugin is loaded` as Then test step since it is testing something (istio-system card is visible) and not a user interaction like click. Check also my comment below about renaming the test step.

Reply:
The page is loaded chronologically like this, first the overview card with heading, than it loads the cpu and memory charts via different request. Since charts are last elements to load, I think its more reasonable to have it as final assertion.